rnd/autogpt_libs/autogpt_libs/supabase_integration_credentials_store/types.py

```diff
@@ -38,9 +38,17 @@ class APIKeyCredentials(_BaseCredentials):
 ]
 
 
+class OAuthState(BaseModel):
+    token: str
+    provider: str
+    expires_at: int  # Unix timestamp
```

Comment:
Maybe better make it `"""docstring"""` or even `Field` with description

Reply:
you wrote it lol

Reply:
Claude*, I had a feeling this was AI comment 😂 but there's more similar with e.g. `# seconds`